cmd/kyma/apply/function/function.go

```diff
@@ -99,12 +95,17 @@ func (c *command) Run() error {
 		operators[gitOperator] = nil
 	}
 
+	operators[operator.NewGenericOperator(client.Resource(operator.GVKFunction).Namespace(configuration.Namespace), function)] = []operator.Operator{
+		operator.NewTriggersOperator(client.Resource(operator.GVKTriggers).Namespace(configuration.Namespace), triggers...),
+	}
+
 	mgr := manager.NewManager(operators)
 	options := manager.Options{
 		Callbacks:          callbacks(c),
 		OnError:            chooseOnError(c.opts.OnError),
 		DryRun:             c.opts.DryRun,
 		SetOwnerReferences: true,
+		WaitForApplying:    true,
```

Comment:
Shouldn't this be configurable per flag?

I would also call it `watch` to be consistent with other CLI commands and tools like `kubectl`.
See:
https://github.com/kyma-project/cli/blob/33e45cb9ff60fc2b80f502873618628ea2feb425/cmd/kyma/test/run/cmd.go#L60

Reply:
There will be more changes so I think that this PR is not ready to review yet. Of course, I will apply your suggestion but in the next commits, I will add more changes ;)